motion_udp/JointData position
motion_udp/JointData velocity
motion_udp/JointData torque
